packages/ethereum-contracts/contracts/agreements/AgreementForwarder.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity >= 0.8.0;
+
+
+/**
+* Provides an easier to use interface to Superfluid agreement specific functionality of Super Tokens
+*/
+
+import {
+    ISuperfluid,
+    ISuperToken,
+    BatchOperation
+} from "../interfaces/superfluid/ISuperfluid.sol";
+
+import {
+    IConstantFlowAgreementV1
+} from "../interfaces/agreements/IConstantFlowAgreementV1.sol";
+
+import { CallUtils } from "../libs/CallUtils.sol";
+
+// TODO: consider adding meta-tx capabilities to all functions
+contract AgreementForwarder {
+
+    ISuperfluid internal _host;
+    IConstantFlowAgreementV1 internal _cfa;
+
+    constructor(ISuperfluid host) {
+        _host = host;
+        _cfa = IConstantFlowAgreementV1(address(_host.getAgreementClass(
+            keccak256("org.superfluid-finance.agreements.ConstantFlowAgreement.v1")
+        )));
+    }
+
+    function createConstantFlow(
```

Comment:
oh no, no more short hand `createFlow` :) ?

I don't mind, I would change the spec too.